src/app/campaign/store/reducers/flight.reducer.ts

```diff
@@ -92,7 +90,12 @@ export function reducer(state = initialState, action: CampaignActions): State {
     }
     case ActionTypes.CAMPAIGN_FLIGHT_FORM_UPDATE: {
       const { flight, changed, valid } = action.payload;
-      const localFlight = { ...(state.entities[flight.id] && state.entities[flight.id].localFlight), ...flight };
+      const localFlight = {
+        ...(state.entities[flight.id] && state.entities[flight.id].localFlight),
+        ...flight,
+        dailyMinimum: flight.dailyMinimum || 0,
```

Comment:
I'm still a bit confused where default values are set - some overlap with the form setting defaults, the reducer, and the actions.  Is there any standard place to do that in reactive forms, or more of an "everybody set their own defaults" belt-and-suspenders thing?

Reply:
There is a standard place for the initial values in reactive forms when you set them up with the form builder, but not defaults as in if a field is not set in a form update, default to this value.

We do have these fields initialized https://github.com/PRX/grove.prx.org/pull/200/files#diff-51fa2200ec802e5c3d1a18c7dc52a7a1R83, but undefined/null/empty values were getting thru.